### DISCUSSION TITLE
Issues no longer loading?

### DISCUSSION THREAD

**Discussion by darshanbib** [community/community#52833] [Projects and Issues]
### Select Topic Area

Bug

### Body

Clicking on an issue no longer loads any details. You have to click Open in a new tab to load the issue. What is the purpose of having it work this way?
<img width="1248" alt="Screenshot 2023-04-14 at 12 35 30 PM" src="https://user-images.githubusercontent.com/112594/232104310-8d42839d-3247-49c7-a3cf-82322e741f82.png">

Only after clicking the the "Open in new tab" link can you see the info.

**Comment by abstractalgo** [comment]
~~+1 Experiencing the same problem.~~

All good now.

**Comment by Loubz** [comment]
Having the same problem with cards and Issues from Projects (only in Chrome, tried Incognito mode and it does the same thing, works ok in Firefox).
